R/pkg_ref_cache_remote_checks.R

```diff
@@ -0,0 +1,45 @@
+#' Retrieve a CRAN or Bioc checks or run R CMD check
+#'
+#' @inheritParams pkg_ref_cache
+#' @family package reference cache
+#'
+pkg_ref_cache.cran_checks <- function (x, ...)
+{
+   UseMethod("pkg_ref_cache.cran_checks")
+}
+
+pkg_ref_cache.cran_checks.default <- function (x, ...) {
+  return(NA)
+}
+
+#' @importFrom xml2 read_html xml_find_all xml_text
+pkg_ref_cache.cran_checks.pkg_cran_remote <- function(x, ...) {
+  URLbase <- "https://cran.r-project.org/web/checks/check_results_"
+  webURL <- paste0(URLbase, x$name, ".html")
+
+  page <- read_html(webURL)
```

Comment:
We can pull this in before we look toward testing it, but eventually we'll need to replace `read_html` with `httr::content(httr:GET(...))` in order to intercept web requests with `webmockr`.